docs/feature-service-support.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: ArcGIS Feature Service Support
+nav_order: 4
+---
+
+The MarkLogic Koop Provider supports multiple operations in the ArcGIS Feature Service API, each of which is detailed
+below.
+
+## Query Layer Operation
+
+The following parameters in the
+[Query Layer operation](https://developers.arcgis.com/rest/services-reference/enterprise/query-feature-service-layer-.htm)
+are supported (these are listed in the order they appear in the ArcGIS documentation):
+
+- `where`
```

Comment:
I was thinking - perhaps we describe each of these - but the link above already does that, and I don't know of anything worth adding for any of these that the ArcGIS docs don't already state.

Reply:
yeah, there's nothing more to add. Additionally, if anything changes on that page, we'd have to change it here.